src/Service/MediaConvert/tests/Unit/MediaConvertClientTest.php

```diff
@@ -57,14 +60,7 @@ public function testCreateJob(): void
 
     public function testDescribeEndpoints(): void
     {
-        $client = new MediaConvertClient([], new NullProvider(), new MockHttpClient());
-
-        $input = new DescribeEndpointsRequest([
-        ]);
-        $result = $client->describeEndpoints($input);
-
-        self::assertInstanceOf(DescribeEndpointsResponse::class, $result);
-        self::assertFalse($result->info()['resolved']);
+        self::markTestSkipped('method is deprecated');
```

Comment:
a deprecation should not skip the test. You should keep the existing test but mark it as `@group legacy`. Otherwise, we loose code coverage for the deprecated code.

Reply:
thanks